common/bigquery/Makefile

```diff
@@ -1,29 +1,40 @@
 # Makefile for BigQuery modules
 
+CLOUD = bigquery
+
 SED = sed
 GSUTIL = gsutil
 BQ = bq --location=$(BQ_REGION) --project_id=$(BQ_PROJECT)
 
+BQ_MODULE_LABEL ?= spatial_extension_module:core
+BQ_DEPLOY_DATASET = $(BQ_DATASET_PREFIX)carto
 BQ_DATASET = $(BQ_DATASET_PREFIX)$(MODULE)
```

Comment:
Could we rename this? (I assume it won't be used as a dataset name, right?) perhaps BQ_DEPLOY_MODULE ?

Reply:
BQ_DEPLOY_DATASET is the dataset where the toolbox is installed.

Reply:
I have changed `$(BQ_DATASET_PREFIX)$(MODULE)` to be used directly in the bucket variable